posthog/models/dashboard.py

```diff
@@ -23,7 +27,13 @@ class Dashboard(models.Model):
     last_accessed_at: models.DateTimeField = models.DateTimeField(blank=True, null=True)
     filters: models.JSONField = models.JSONField(default=dict)
     creation_mode: models.CharField = models.CharField(max_length=16, default="default", choices=CREATION_MODE_CHOICES)
-    tags: ArrayField = ArrayField(models.CharField(max_length=32), blank=True, default=list)
+
+    global_tags: GenericRelation = GenericRelation(EnterpriseTaggedItem, related_query_name="dashboard")
```

Comment:
Any particular reason we're using `related_query_name` instead of `related_name`